packages/spruce-next-helpers/src/skillskit/helpers/lang.ts

```diff
@@ -0,0 +1,115 @@
+import * as moment from 'moment'
+import Dinero from 'dinero.js'
+
+import { ICoreLocation } from '@sprucelabs/spruce-types'
+
+export default {
+	lang: {},
+	overrides: {},
+	async configure(langDir: string) {
```

Comment:
As far as making this more generic, what about passing in `lang` and `overrides` as objects, and defer to the individual usage to require and provide the data, rather than `require`ing here? Would also make this more testable I think.